### PR TITLE
Update apache-project-maturity-model.md

### PR DESCRIPTION
**Pull request by cottage14**
Proposed edits to improve readability. Passive voice can obscure who does what, and people whose primary language is not English can find it hard to understand.

### CODE REVIEW

File: source/apache-way/apache-project-maturity-model.md
Comment (adityasharma7) [inline source/apache-way/apache-project-maturity-model.md:21]:
```suggestion
We welcome Questions and feedback about this model on the <a href="https://lists.apache.org/list.html?dev@community.apache.org" target="_blank">comdev mailing list</a>.
```

**Comment by bdelacretaz** [comment]
Thank you for this! Given that this is a somewhat normative document and not just prose, I suggest making the dev@community.a.o list aware of these changes before merging them. I could drive that but I probably won't have time before early January, so if you don't mind letting this sit for a while I would appreciate it.

File: source/apache-way/apache-project-maturity-model.md
Comment (clr-apache) [inline source/apache-way/apache-project-maturity-model.md:110]: Releases are signed, and distributed along with digests and signatures that anyone can reliably use to validate the downloaded archives.

File: source/apache-way/apache-project-maturity-model.md
Comment (clr-apache) [inline source/apache-way/apache-project-maturity-model.md:215]: Decisions require consensus among PMC members

File: source/apache-way/apache-project-maturity-model.md
Comment (clr-apache) [inline source/apache-way/apache-project-maturity-model.md:378]: for example to discuss and to vote on PMC and committer candidates.

**Review by clr-apache — CHANGES_REQUESTED** [review]
With these changes (typos) this PR is good to go. It does not change anything substantial but simply makes the document more readable.

File: source/apache-way/apache-project-maturity-model.md
Comment (sebbASF) [reply source/apache-way/apache-project-maturity-model.md:215]: reqyuire?

**Comment by bdelacretaz** [comment]
I created #61 based on this PR as (I think) I couldn't update this one as it's based on a different repository.

I have committed the resulting changes, many thanks to @cottage14 and all who provided input here!

The resulting commit is https://github.com/apache/comdev-site/commit/bed7a048a23ced862b654804f2c50d7dba56e9a0

This PR can now be closed, if more changes are needed we can create more specific ones. I don't seem to be able to close it, @cottage14 can you do that if you agree?

**Comment by swapnilmmane** [comment]
Thank you @bdelacretaz!

**Pull request merged**
